packages/swarm-docs/src/pages/Radio.mdx

```diff
@@ -1,12 +1,62 @@
-import { Radio } from '@meetup/swarm-components'
-import Example from '../components/radioExample'
+import { Radio } from '@meetup/swarm-components';
+import Example, { GroupRadioButtonsExample } from '../components/radioExample';
+import PropsTable from '../components/propstable';
 
-# Radio
+## Radio
+---
+A wrapper for an html `<input type="radio">`.
+
+<br/>
+<br/>
+
+### Snippet
+---
+
+Use this interactive snippet to try the component:
+<br/>
 
 ```.jsx
-<Radio label="swarm" checked={1 == 1} />
+<Radio name="swarm" label="swarm" value="1" checked={1 == 1} />
```

Comment:
What throws a console error about lack of `onChange` handler is this example that doesn't have one.

Reply:
done